src/CTA.WebForms2Blazor/ControlConverters/ControlConverter.cs

```diff
@@ -21,22 +28,80 @@ public virtual HtmlNode Convert2Blazor(HtmlNode node)
             return Convert2BlazorFromParts(NodeTemplate, BlazorName, ConvertAttributes(node.Attributes), node.InnerHtml);
         }
 
-        private string ConvertAttributes(HtmlAttributeCollection attributeCollection)
+        protected string ConvertAttributes(HtmlAttributeCollection attributeCollection)
         {
             var convertedAttributes = attributeCollection
                 .Where(attr => AttributeMap.ContainsKey(attr.Name))
-                .Select(attr => $"{AttributeMap[attr.Name]}={attr.Value}");
-            
+                .Select(attr => $"{AttributeMap[attr.Name]}='{attr.Value}'");
+
             var convertedAttributesString = string.Join(" ", convertedAttributes);
             
             return convertedAttributesString;
         }
 
-        private HtmlNode Convert2BlazorFromParts(string template, string name, string attributes, string body)
+        protected HtmlNode Convert2BlazorFromParts(string template, string name, string attributes, string body)
```

Comment:
Is there ever a time when template won't be NodeTemplate? If not maybe we should get rid of the parameter and just use that

Reply:
In LabelConverter I changed the template to be just the text value, but I'm not sure how common thats gonna be